server.js

```diff
@@ -2,21 +2,15 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 
-const environment = process.env.NODE_ENV || 'development';
-const configuration = require('./knexfile')[environment];
-const database = require('knex')(configuration);
-
 const FoodsController = require('./lib/controllers/foods-controller')
+const Food = require('./lib/models/food')
```

Comment:
Let's remember to remove this line since we don't need the Food model in server anymore.